.ado/templates/react-native-init.yml

```diff
@@ -70,7 +70,7 @@ steps:
   - script: yarn build
     displayName: yarn build
 
-  - powershell: start-process npx -ArgumentList @('verdaccio', '--config', './.ado/verdaccio/config.yaml')
+  - powershell: start-process verdaccio.cmd -ArgumentList @('--config', './.ado/verdaccio/config.yaml')
```

Comment:
nit: maybe also `yarn verdaccio` to keep the illusion of platform independent script? :D

Reply:
I didn't dig too deeply, but for some reason or another `npx` had trouble finding the packages I had globally installed via the managed image artifact I used to install them. I think it installs them to a non-standard directory maybe?

An advantage though is making it clear we are using a system installed version of the package, instead of something present as an RNW dependency.